Model/Cron.php

```diff
@@ -32,9 +35,16 @@
 use Magento\Framework\Phrase\Renderer\Placeholder;
 use Magento\Framework\Phrase;
 use Magento\Sales\Model\OrderRepository;
+use Magento\Vault\Api\Data\PaymentTokenFactoryInterface;
+use Magento\Vault\Api\PaymentTokenRepositoryInterface;
+use Magento\Vault\Model\PaymentTokenManagement;
 
 class Cron
 {
+    const RECURRING_DETAIL_REFERENCE = 'recurring.recurringDetailReference';
+    const EXPIRY_DATE = 'expiryDate';
+    const PAYMENT_METHOD = 'paymentMethod';
```

Comment:
These three could be added to and reused from the `Adyen\Payment\Helper\Vault` class

Reply:
Done